src/transformation/visitors/call.ts

```diff
@@ -322,7 +322,9 @@ export const transformCallExpression: FunctionVisitor<ts.CallExpression> = (node
     if (!isContextualCall) {
         [callPath, parameters] = transformCallAndArguments(context, node.expression, node.arguments, signature);
     } else {
-        const callContext = context.isStrict ? ts.factory.createNull() : ts.factory.createIdentifier("_G");
+        // if is optionalContinuation, context will be handled by transformOptionalChain.
+        const callContext =
+            optionalContinuation || context.isStrict ? ts.factory.createNull() : ts.factory.createIdentifier("_G");
```

Comment:
Shouldn't this be a `??`

Reply:
It's a boolean check, so if anything, the proper way to do it would be `optionalContinuation !== undefined || context.isStrict ? ...`

Reply:
Its equivalent to || in this case, and its only used as a condition of ternary. Should we still change it?